go/engine/pgp_import_key.go

```diff
@@ -332,7 +332,12 @@ func (e *PGPKeyImportEngine) prepareSecretPush(ctx *Context) error {
 func (e *PGPKeyImportEngine) push(ctx *Context) (err error) {
 	e.G().Log.Debug("+ PGP::Push")
 	if e.arg.GPGFallback {
-		e.bundle.InitGPGKey()
+		e.bundle.GPGFallbackKey = libkb.NewGPGKey(
+			nil,
```

Comment:
It wants `*GlobalContext` here, and it calls `Contextified: NewContextified(g)` with it, but it seems fine? Do I have a GlobalContext to pass here?